examples/ocean-heat-content/ocean-heat-content-1972-03-2000m/ocean-heat-content-1972-03-2000m.json

```diff
@@ -59,6 +73,15 @@
       "href": "./heat_content_anomaly_0-2000_monthly_1972-03.tif",
       "type": "image/tiff; application=geotiff; profile=cloud-optimized",
       "title": "Ocean Heat Content anomalies from WOA09 : heat_content_anomaly 0-2000 m monthly 1.00 degree 1972-03",
```

Comment:
Might be worth trying to make the titles more readable/understandable.

Reply:
I pull this title directly from the NetCDF, so this falls into the "how much extra processing do" categories of issues. I've updated to be _a bit_ clearer, LMK what you think.

Reply:
I take your point. I'm guessing it's not an easy fix with so many titles. I'm not going to lose sleep over it.